.circleci/config.yml

```diff
@@ -609,30 +622,36 @@ jobs:
             - golang-build-cache-contracts-bedrock-tests-{{ checksum "go.sum" }}
             - golang-build-cache-contracts-bedrock-tests-
       - run:
-          name: print dependencies
+          name: Print dependencies
           command: just dep-status
           working_directory: packages/contracts-bedrock
       - run:
-          name: print forge version
+          name: Print forge version
           command: forge --version
           working_directory: packages/contracts-bedrock
       - run:
           name: Pull artifacts
           command: bash scripts/ops/pull-artifacts.sh
           working_directory: packages/contracts-bedrock
       - run:
-          name: build go-ffi
+          name: Build go-ffi
           command: just build-go-ffi
           working_directory: packages/contracts-bedrock
       - run:
-          name: run tests
-          command: <<parameters.test_command>>
+          name: Run tests
+          command: |
+            TEST_FILES=$(<<parameters.test_list>>)
+            TEST_FILES=$(echo "$TEST_FILES" | circleci tests split --split-by=timings)
+            TEST_FILES=$(echo "$TEST_FILES" | sed 's|^test/||')
+            MATCH_PATH="./test/{$(echo "$TEST_FILES" | paste -sd "," -)}"
+            export FOUNDRY_INVARIANT_RUNS=<<parameters.test_fuzz_runs>>
```

Comment:
Invariant tests are much slower than regular fuzz tests, I don't think we want to use the same amount of runs for both. For invariant tests it's better to have fewer test runs with a very large depth (longer call sequence). See https://github.com/foundry-rs/foundry/issues/7848